app/experimenter/experiments/bugzilla.py

```diff
@@ -38,13 +53,52 @@ def format_bug_body(experiment):
             ]
         )
         bug_body = experiment.BUGZILLA_PREF_TEMPLATE.format(
-            experiment=experiment, variants=variants_body
+            experiment=experiment,
+            variants=variants_body,
+            countries=countries,
+            locales=locales,
         )
 
     return bug_body
 
 
-def make_bugzilla_call(url, data):
+def format_update_body(experiment):
+    cf_tracking = "cf_tracking_firefox{}".format(
+        get_firefox_major_version(experiment.firefox_version)
+    )
+    summary = "[Experiment] {experiment_name} Fx {version} {channel}".format(
+        experiment_name=experiment,
+        version=experiment.firefox_version,
+        channel=experiment.firefox_channel,
+    )
+    return {
+        "summary": summary,
+        "cf_user_story": format_bug_body(experiment),
+        "whiteboard": experiment.STATUS_SHIP_LABEL,
+        cf_tracking: "?",
```

Comment:
bugzilla's api ignores invalid fields when doing a PUT, so if cf_tracking version is missing or the version is not a valid option, the rest call will still be made successfully without it. Not sure if this is okay behaviour or should we check and notify user that cf_tracking will not be updated due it either being missing or not an option in bugzilla?

Reply:
Ideally we wanna know if it fails and raise some kind of error to go to sentry and notify the user, but if we don't get any signals back about what failed then there's not much we can do.